.solhint.json

```diff
@@ -2,9 +2,14 @@
   "extends": "solhint:recommended",
   "plugins": ["prettier"],
   "rules": {
+    "no-unused-import": "error",
```

Comment:
This ensures that it will fail in the future

Reply:
lovely